assets/auto-vj-uri.html

```diff
@@ -0,0 +1,141 @@
+<html>
+    <head>
+        <title>AutoVJ</title>
+        <script type="text/javascript" src="api/api-ws.js"></script> 
+        <script type="text/javascript" src="api/api.js"></script>
+        <script type="text/javascript">
+
+            /* Place the URI to where to obtain videos.
+               Do not use rouille /assets folder as it doesn't supports ranges which is required for setting the video current time (syncing/seek).
```

Comment:
While this is true, not loading the whole file in-memory prior to playing it also makes it susceptible to lag during playback or seeking. 

Could work in an isolated LAN environment, but I'd not expect most clubs to have anything but crappy WiFi broadcasted by the router hanging by it's WAN wire half soaked in the toilet bowl, so... tldr use with extreme caution :-)

Reply:
> While this is true, not loading the whole file in-memory prior to playing it also makes it susceptible to lag during playback or seeking.
> 
> Could work in an isolated LAN environment, but I'd not expect most clubs to have anything but crappy WiFi broadcasted by the router hanging by it's WAN wire half soaked in the toilet bowl, so... tldr use with extreme caution :-)

Correct! Best case scenario is to have the web server streaming the videos in the same device running 'traktor-obs-relay'.
This was a must for me because when downloading the whole video as a blob into memory, even if the media web server was local, it was taking around 3 seconds or even more with heavy files above 300Mb, and like this it loads instantly.

Reply:
Interesting. I've been using rouille and things have been instant with files up to 1Gb or so when loading as a blob into an OBS chromium frame. 

Either way this discussion is purely for documentation/history, not code feedback.

Reply:
> Interesting. I've been using rouille and things have been instant with files up to 1Gb or so when loading as a blob into an OBS chromium frame.
> 
> Either way this discussion is purely for documentation/history, not code feedback.

This is true for fast computers but I had to test with a very basic computer and loading just the initial range chunk and streaming the file (nginx default behaviour) works much better. This also helps crappy WiFi scenarios when the media server is in the network instead.